main/nucleus/test-utils/utils-ng/src/main/java/org/glassfish/tests/utils/NucleusTestUtils.java

```diff
@@ -335,4 +335,58 @@ public static String getURL(String urlstr) {
         public String outAndErr;
     }
 
+    public static void deleteDirectoryContents(final File dir) {
```

Comment:
The method is unused but might be of use later, in case the current workaround of touch does not work, its intermittent so just added for future purposes.